.pre-commit-config.yaml

```diff
@@ -9,3 +9,7 @@ repos:
       - id: check-yaml
       - id: check-json
       - id: check-added-large-files
+  - repo: https://github.com/ucb-bar/checkmake.git
```

Comment:
Why use a fork instead of upstream?

Reply:
The upstream checks if the makefile contains a bunch of default PHONYs & we don't need that.

Reply:
I suspect there's a way to pass args to the pre-commit hook defined in that repo. https://pre-commit.com/#new-hooks

Is there a way to pass a arg that disables the min-phony stuff?

Reply:
Done!